spec/support/forms/mark_for_destruction_form.rb

```diff
@@ -0,0 +1,21 @@
+class MarkForDestructionForm < YAAF::Form
```

Comment:
```suggestion
class UserDestroyForm < YAAF::Form
```

minor comment, feel free to ignore